src/MoBi.Presentation/Tasks/OutputSelectionsTask.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Linq;
+using MoBi.Assets;
+using MoBi.Presentation.Presenter;
+using OSPSuite.Core.Commands;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Repositories;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Core.Services;
+
+namespace MoBi.Presentation.Tasks
+{
+   public interface IOutputSelectionsTask
+   {
+      /// <summary>
+      /// Adds a new output selection to the <paramref name="simulationSettings"/>. The <paramref name="preSelectedQuantitySelection"/>
+      /// is preselected in the tree view
+      /// </summary>
+      void AddOutputSelection(SimulationSettings simulationSettings, QuantitySelection preSelectedQuantitySelection = null);
+
+      /// <summary>
+      /// Edit the <paramref name="selection"/> in the <paramref name="simulationSettings"/> using the interactive editor
+      /// </summary>
+      void EditOutputSelection(SimulationSettings simulationSettings, QuantitySelection selection);
+
+      /// <summary>
+      /// Removes the <paramref name="selection"/> from the <paramref name="simulationSettings"/>
+      /// </summary>
+      void RemoveOutputSelection(SimulationSettings simulationSettings, QuantitySelection selection);
+
+      /// <summary>
+      /// Updates the <paramref name="selection"/> with the <paramref name="newPath"/>
+      /// </summary>
+      void UpdateOutputSelection(SimulationSettings simulationSettings, QuantitySelection selection, string newPath);
+   }
+
+   public class OutputSelectionsTask : IOutputSelectionsTask
+   {
+      private readonly IOSPSuiteExecutionContext _context;
+      private readonly IMoBiApplicationController _applicationController;
+      private readonly ISimulationRepository _simulationRepository;
+      private readonly IDialogCreator _dialogCreator;
+
+      public OutputSelectionsTask(IOSPSuiteExecutionContext context, IMoBiApplicationController applicationController, ISimulationRepository simulationRepository, IDialogCreator dialogCreator)
+      {
+         _context = context;
+         _applicationController = applicationController;
+         _simulationRepository = simulationRepository;
+         _dialogCreator = dialogCreator;
+      }
+
+      public void AddOutputSelection(SimulationSettings simulationSettings, QuantitySelection preSelectedQuantitySelection = null)
+      {
+         var pathFromSimulations = selectPathFromSimulationsFor(AppConstants.Captions.AddDefaultCurveForNewSimulations, preSelectedQuantitySelection);
+         if(pathFromSimulations == null)
+            return;
+
+         if (!validateAndWarnNewPath(pathFromSimulations, simulationSettings))
+            return;
+
+         simulationSettings.OutputSelections.AddOutput(new QuantitySelection(pathFromSimulations, QuantityType.Undefined));
```

Comment:
```QuantityType.Undefined```? I don't know about this. The quantity type is not needed for this view, but I think it's still required in the domain object.

When I tested, the simulation view shows the correct QuantityTypes after the simulation is created

Reply:
hum this will be used later to filter match entries by path etc when using the chart template
Can't you find it from the selection? (e.g. pathFromSimulations => instead of string, path and quantitytype?)

Reply:
Yeah, probably, but it could be hand edited after as well. When I use this default output selection to create new simulations, then they look correct in the simulation settings. 

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/261477/9246a230-40db-431f-94b4-412a874cf328)